Assets/Scripts/GameOverManager.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+
+
+public class GameOverManager : MonoBehaviour
+{
+    GameObject[] racerList;
+    List<GameObject> finishedRacers;
+    // Start is called before the first frame update
+    void Start()
+    {
+        racerList = GameObject.FindGameObjectsWithTag("Racer");
```

Comment:
Do not use FindGameObjectsWithTag. It is very slow and we already have a RacerManager to fetch the Rigidbodies of every racer in the scene. You can use those rigidbodies to access their game objects instead which would be much faster.